checkstyle-tester/diff.groovy

```diff
@@ -161,6 +135,20 @@ def hasUnstagedChanges(gitRepo) {
     return hasUnstagedChanges
 }
 
+def runPlainCheckstyleReportsGeneration(cfg) {
+    generateCheckstyleReport(cfg.localGitRepo, cfg.patchBranch, cfg.patchCheckstyleCfg, cfg.listOfProjects, cfg.reportsDir)
+    generateSummaryIndexHtml(cfg.reportsDir)
```

Comment:
@MEZk In the end, I only want to look at diff reports and not the summary report. :)
I think our report is alot easier to view.

If we made `patch-diff-report-tool` support no base config or base reports, we wouldn't need this extra parsing or different code for the summary index. Then all reports would look the same and it would be easier to make more additions to the summary, if we decide to.

Does this sound like a better idea?
@romani What do you think?